src/MainWindow.cpp

```diff
@@ -840,7 +840,8 @@ void MainWindow::doubleClickTable(const QModelIndex& index)
     editDock->allowEditing(allowEditing);
 
     // Load the current value into the edit dock
-    editDock->loadText(index.data(Qt::EditRole).toByteArray(), index.row(), index.column());
+    editDock->loadDataFromCell(index.data(Qt::EditRole).toByteArray(),
+                               index.row(), index.column());
```

Comment:
Trying to keep it within roughly 80 chars.  And yeah, other places could use clean-up too. :smile: